data/tausiq2003.json

```
@@ -0,0 +1,109 @@
+{
+  "name": "Tausiq Samantaray",
+  "type": "personal",
+  "bio": "CS Student | Major focus: **Web dev** and **software dev** | Exploring different technologies",
+  "tags": [
+    "Open Source",
+    "Python",
+    "Web Dev",
+    "Java"
+  ],
+  "socials": [
+    { "icon": "FaTwitter", "url": "https://twitter.com/tausiqtweets" },
+    { "icon": "FaGithub", "url": "https://github.com/tausiq2003" },
+    { "icon": "FaLinkedIn", "url": "www.linkedin.com/in/tausiq-samantaray" }
+  ],
+  "avatar": "https://github.com/tausiq2003.png",
+  "links": [
+    {
+      "group": "Socials",
+      "name": "Twitter",
+      "url": "https://twitter.com/tausiqtweets",
+      "icon": "FaTwitter"
+    },
+    {
+      "group": "Socials",
+      "name": "Youtube",
+      "url": "https://www.youtube.com/@tausiqsamantaray3363/",
+      "icon": "FaYoutube"
+    },
+    {
+      "group": "Socials",
+      "name": "Github",
+      "url": "https://github.com/tausiq2003",
+      "icon": "FaGithub"
+    },
+    {
+      "group": "Socials",
+      "name": "LinkedIn",
+      "url": "www.linkedin.com/in/tausiq-samantaray",
+      "icon": "FaLinkedIn"
```

You probably be using an invalid icon. It should be `FaLinkedin`.
Also, add `https://` extension at the beginning of your URL because it may use the host domain later. 

```suggestion
      "url": "https://www.linkedin.com/in/tausiq-samantaray",
      "icon": "FaLinkedin"
```